src/main/resources/standard_files/.gitignore

```diff
@@ -0,0 +1,81 @@
+##############################
```

Comment:
Müssen wir aufpassen, dass die gitignore nicht auch noch Sachen in unserem Generatorprojekt blockiert, nachdem die gitignore generiert wurde? Ich weiß nicht ob die in der Ordnerstruktur Level nach oben geht

Reply:
Fair. Ich benenne die "template" gitignore im Generatorprojekt einfach um :)